### DISCUSSION TITLE
Upgrade Software Stop Working

### DISCUSSION THREAD

**Discussion by Kuroo2021** [aclap-dev/video-downloadhelper#749] [Help and Q&A]
### Your question

After the addon was upgraded it seems to stop working on many sites and just shows a blank white box.

### Addon & CoApp version and details

This information doesn't even pop up when I click on the add on the gear icon doesn't even show up as well. I was able to fine the version Version 9.0.1.22, anytime before this it worked fine.

**Comment by paulrouget** [comment]
What is your browser and what is the version of the browser you use?

**Reply by Kuroo2021** [reply]
> What is your browser and what is the version of the browser you use?

Google Chrome Version 124.0.6367.208 (Official Build) (64-bit)

**Reply by samuto** [reply]
Me too, plugin no longer working, I am really keen to roll back to the previous version, even with the Firefox latest update, it still not work well. I prefer the previous UI, and I am able to to click those small colour circle to decide which one to download. The latest one only display the ad videos on the site which I don't need, and those useful video links are all disapeared.

**Reply by samuto** [reply]
reversed back to version 8, now works :)

https://addons.mozilla.org/firefox/downloads/file/4251369/video_downloadhelper-8.2.2.8.xpi

**Reply by Kuroo2021** [reply]
> reversed back to version 8, now works :)
> 
> https://addons.mozilla.org/firefox/downloads/file/4251369/video_downloadhelper-8.2.2.8.xpi

I'm on Chrome tho.

**Reply by Demodian** [reply]
One reason I think some of these new installs may fail is if the ffmpeg that is bundled does not match the OS level.  I had to hack the 2.0.10 coapp install on my Ubuntu 20.04 Raspberry Pi system.  I'm not running 22.04 or 24.04, so my library versions are older and incompatible with the default bundled binaries.  Yes, it is all ARM. Yes, it is Linux.  No, it's not the same library version.  Maybe the installer needs to be smart about that detection.  I've rolled back to 8, because I don't have time to fight the versioning issue.

**Reply by Kuroo2021** [reply]
> One reason I think some of these new installs may fail is if the ffmpeg that is bundled does not match the OS level. I had to hack the 2.0.10 coapp install on my Ubuntu 20.04 Raspberry Pi system. I'm not running 22.04 or 24.04, so my library versions are older and incompatible with the default bundled binaries. Yes, it is all ARM. Yes, it is Linux. No, it's not the same library version. Maybe the installer needs to be smart about that detection. I've rolled back to 8, because I don't have time to fight the versioning issue.

No clue but I run Windows 11 Pro
